### ISSUE TITLE
Unable to get nbpapaya working

### ISSUE THREAD

**Issue by abhijithchunduru** [akeshavan/nbpapaya#8]
Hi, 
I cloned the repo, downloaded all the dependent git repos and did a python setup.py install. And placed papaya.js and papaya.css in ~/.jupyter/custom and ~/.ipython/profile_default/static/custom. I keep getting "404 page not found". I want to install nbpapaya on a server which has a lot of nifti images and go through them on a client. Therefore I start my jupyter with jupyter notebook --ip=<<ip address>>. 
Will appreciate if I get some tips on debugging and installing nbpapaya properly

**Comment by kastman** [comment]
Did you remember to pull in Papaya as a submodule with ``git submodule update --init --recursive`` like it says in the readme? Papaya isn't available without it.

**Comment by abhijithchunduru** [comment]
Yes, I did. And also ensured papaya.css and papaya.js are in .ipython and .jupyter folders

**Comment by bfortuner** [comment]
Had the same issue. Did you figure this out?

**Comment by augix** [comment]
got the same error message when running Brain("./nbpapaya/Papaya/data/sample_image.nii.gz") in example.ipynb: "404 : Not Found. You are requesting a page that does not exist!"
